lib/features.js

```diff
@@ -18,75 +18,106 @@ const packageHelper = require('./package-helper');
 const features = {
     sass: {
         method: 'enableSassLoader()',
-        packages: ['sass-loader', 'node-sass'],
-        description: 'load SASS files'
+        packages: [
+            { name: 'sass-loader', version: 7 },
+            { name: 'node-sass' }
+        ],
+        description: 'load Sass files'
     },
     less: {
         method: 'enableLessLoader()',
-        packages: ['less-loader'],
+        packages: [
+            { name: 'less-loader', version: 4 },
+        ],
         description: 'load LESS files'
     },
     stylus: {
         method: 'enableStylusLoader()',
-        packages: ['stylus-loader', 'stylus'],
+        packages: [
+            { name: 'stylus-loader', version: 3 },
+            { name: 'stylus' }
+        ],
         description: 'load Stylus files'
     },
     postcss: {
         method: 'enablePostCssLoader()',
-        packages: ['postcss-loader'],
+        packages: [
+            { name: 'postcss-loader', version: 2 }
+        ],
         description: 'process through PostCSS'
     },
     react: {
         method: 'enableReactPreset()',
-        packages: ['babel-preset-react'],
+        packages: [
+            { name: '@babel/preset-react', version: 7 }
+        ],
         description: 'process React JS files'
     },
     preact: {
         method: 'enablePreactPreset()',
-        packages: ['babel-plugin-transform-react-jsx'],
+        packages: [
+            { name: '@babel/plugin-transform-react-jsx', version: 7 }
+        ],
         description: 'process Preact JS files'
     },
     typescript: {
         method: 'enableTypeScriptLoader()',
-        packages: ['typescript', 'ts-loader'],
+        packages: [
+            { name: 'typescript' },
+            { name: 'ts-loader', version: 4 },
+        ],
         description: 'process TypeScript files'
     },
-    coffeescript: {
-        method: 'enableCoffeeScriptLoader()',
-        packages: ['coffeescript', 'coffee-loader'],
-        description: 'process CoffeeScript files'
-    },
     forkedtypecheck: {
         method: 'enableForkedTypeScriptTypesChecking()',
-        packages: ['typescript', 'ts-loader', 'fork-ts-checker-webpack-plugin'],
+        packages: [
+            { name: 'typescript' },
+            { name: 'ts-loader', version: 4 },
+            { name: 'fork-ts-checker-webpack-plugin', version: 0 },
```

Comment:
this will ask to install `fork-ts-checker-webpack-plugin@^0.0`, which won't work.

You should not handle `version` as a number IMO, but as a semver version constraint instead IMO (use the `semver` package to check that then).